docs/README.rst

```diff
@@ -344,6 +364,40 @@ To configure health monitors on your Ingress resource, you need to use the appro
     }
 
 
+OpenShift Route Resources
+-------------------------
+The |kctlr-long| supports OpenShift Route resources as an alternative to F5 Resource ConfigMaps and Kubernetes Ingress Resources.
+
+Supported Route Configurations
+``````````````````````````````
+
++-------------------------+-------------------+-------------------+---------+-----------------+-------------------------------------------------------------------------+
+| Type                    | Client Connection | Server Connection | Path    | SSL Termination | Notes                                                                   |
+|                         | Encrypted         | Encrypted         | Support | on BIG-IP       |                                                                         |
++=========================+===================+===================+=========+===========================================================================================+
+| Unsecured               | No                | No                | Yes     | No              | The BIG-IP system forwards unsecured traffic from the client to the     |
+|                         |                   |                   |         |                 | endpoint.                                                               |
++-------------------------+-------------------+-------------------+---------+-----------------+-------------------------------------------------------------------------+
```

Comment:
When viewing this file there seems to be some formatting issues in this first section of the table, though looking at the code I can't exactly tell what is causing it.

Reply:
I found it, will fix.